modules/Emails/include/ComposeView/ComposeView.tpl

```diff
@@ -38,6 +38,21 @@
  * display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
 *}
+{literal}
+    <style>
```

Comment:
This should be in the emails.scss file and not in a style tag. the calc method is supported in SASS